src/routes/v2/dataRetrieval.ts

```diff
@@ -777,6 +777,8 @@ async function pendingTransactions(
       return res.json({ error: "address can not be empty" })
     }
 
+    console.log(`address: ${address}`)
```

Comment:
Can you remove these two console.log statements/

Reply:
Good catch.